tests/test_protocol.py

```diff
@@ -41,11 +36,14 @@ def __repr__(self):
         return '%s.%s(%r, %r)' % (self.__module__, self.__class__.__name__,
                                   self.x, self.y)
 
-    def __cmp__(self, other):
+    def __eq__(self, other):
         if not isinstance(other, Point):
-            return 1
-        else:
-            return cmp((self.x, self.y), (other.x, other.y))
+            return False
+
+        return (self.x, self.y) == (other.x, other.y)
+
+    def __ne__(self, other):
```

Comment:
Nope, just equality. Python 3 is much more bare-bones; there's no `__cmp__()` (you have to use `__gt__()`, `__lt__()`, etc.), and it won't automatically infer `__ne__()` from `__eq__()`.